src/pages/EventDetailsPage/components/EventDetails/index.tsx

```diff
@@ -47,6 +52,30 @@ function EventDetailsComponent(props: EventDetailsComponentProps) {
     canva: { url: canvaURL, label: 'Canva' },
   };
 
+  const renderSignInButton = () => {
+    if (!isOfficer(userContext)) {
```

Comment:
I think we can just disable both signin and rsvp as long as the status isn't ready

Reply:
So whether the user is officer or not doesn't matter? Okay